packages/devtools_app/.vscode/launch.json

```diff
@@ -0,0 +1,35 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "devtools_app",
+            "cwd": "devtools_app",
+            "request": "launch",
+            "type": "dart",
+            "program": "main.dart"
+        },
+        {
+            "name": "memory/default",
+            "cwd": "devtools_app",
+            "request": "launch",
+            "type": "dart",
+            "program": "test/test_infra/scenes/memory/default.stager_app.dart"
+        },
+        {
+            "name": "memory/diff_snapshot",
```

Comment:
can you also add the performance screen stager as a configuration?

Reply:
added